java/test/jmri/jmrit/display/layoutEditor/load/LayoutEditorTest.xml

```diff
@@ -219,50 +219,51 @@
+    <layoutturnout ident="TO2" type="4" hidden="no" disabled="no" disableWhenOccupied="no" continuing="2" xcen="270.0" ycen="250.0" xa="240.0" ya="240.0" xb="300.0" yb="240.0" xc="300.0" yc="260.0" xd="240.0" yd="260.0" ver="1" class="jmri.jmrit.display.layoutEditor.configurexml.LayoutTurnoutXml" turnoutname="IT8" blockname="Yard" connectaname="T4" connectbname="T5" connectcname="T3" connectdname="T2" />
+    <layoutturnout ident="TO3" type="6" hidden="no" disabled="no" disableWhenOccupied="no" continuing="2" xcen="330.0" ycen="250.0" xa="320.0" ya="240.0" xb="360.0" yb="240.0" xc="340.0" yc="260.0" xd="300.0" yd="260.0" ver="1" class="jmri.jmrit.display.layoutEditor.configurexml.LayoutTurnoutXml" turnoutname="IT9" blockname="Yard" connectaname="T5" connectbname="T20" connectcname="T7" connectdname="T3" />
+    <layoutturnout ident="TO4" type="1" hidden="no" disabled="yes" disableWhenOccupied="no" continuing="2" xcen="202.0" ycen="40.0" xa="184.0" ya="40.0" xb="220.0" yb="40.0" xc="220.0" yc="50.0" xd="184.0" yd="40.0" ver="1" class="jmri.jmrit.display.layoutEditor.configurexml.LayoutTurnoutXml" turnoutname="IT1" blockname="North" connectaname="T1" connectbname="T12" connectcname="T10" />
+    <layoutturnout ident="TO5" type="3" hidden="no" disabled="no" disableWhenOccupied="no" continuing="2" xcen="200.0" ycen="100.0" xa="216.13423843904454" ya="92.10246276855469" xb="180.0000015154656" yb="100.0" xc="187.7315216064453" yc="115.79507446289062" xd="219.9999984845344" yd="100.0" ver="1" class="jmri.jmrit.display.layoutEditor.configurexml.LayoutTurnoutXml" turnoutname="IT4" blockname="Yard" connectaname="T9" connectbname="T13" connectcname="T14" />
+    <layoutturnout ident="TO6" type="2" hidden="no" disabled="no" disableWhenOccupied="no" continuing="2" xcen="400.0" ycen="40.0" xa="420.0" ya="40.0" xb="380.0" yb="40.0" xc="380.0" yc="50.0" xd="420.0" yd="40.0" ver="1" class="jmri.jmrit.display.layoutEditor.configurexml.LayoutTurnoutXml" turnoutname="IT3" blockname="East" connectaname="T21" connectbname="T15" connectcname="T17" />
+    <layoutturnout ident="TO8" type="2" hidden="yes" disabled="no" disableWhenOccupied="no" continuing="2" xcen="320.0" ycen="40.0" xa="360.0" ya="40.0" xb="280.0" yb="40.0" xc="300.0" yc="50.0" xd="360.0" yd="40.0" ver="1" class="jmri.jmrit.display.layoutEditor.configurexml.LayoutTurnoutXml" turnoutname="IT2" blockname="North" connectaname="T15" connectbname="T12" connectcname="T11" />
     <tracksegment ident="T1" blockname="West" connect1name="TO4" type1="2" connect2name="TO1" type2="2" dashed="no" mainline="yes" hidden="no" arc="yes" flip="yes" circle="yes" angle="180.0" hideConLines="yes" class="jmri.jmrit.display.layoutEditor.configurexml.TrackSegmentXml" />
-    <tracksegment ident="T2" blockname="Yard" connect1name="TO2" type1="5" connect2name="TO1" type2="4" dashed="no" mainline="yes" hidden="no" arc="no" class="jmri.jmrit.display.layoutEditor.configurexml.TrackSegmentXml" />
```

Comment:
@geowar1 

You say you only added to the load file, but that isn't true.

look at the track segment T2.  In the old version version, it has the attribute arc="no"  In the new version, that attribute is missing.